### PR TITLE
1.0.4 - fips and distroless fixes

### PR DESCRIPTION
**Pull request by deva26**
fixes issues https://github.com/tetratelabs/getistio/issues/30
https://github.com/tetratelabs/getistio/issues/29

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
       "k8s_versions": [
         "1.17",
         "1.18",
-		"1.19"
+		"1.19",
+		"1.20"
```
Comment (mathetake) [inline manifest.json:17]: nit: indent is broken

File: manifest.json [hunk 2]
```diff
@@ -23,11 +24,12 @@
...
       "version": "1.9.0",
       "flavor": "tetratefips",
-      "flavor_version": 0,
+      "flavor_version": 1,
       "k8s_versions": [
         "1.17",
         "1.18",
-		"1.19"
+		"1.19",
+		"1.20"
```
Comment (mathetake) [inline manifest.json:32]: same here

File: manifest.json [hunk 3]
```diff
@@ -41,7 +43,8 @@
...
```
Comment (mathetake) [inline manifest.json:47]: same here

**Review by mathetake — APPROVED** [review]

File: manifest.json (hunk 1 above)
Comment (deva26) [reply manifest.json:17]: oops. Didn't realize that. 🤦

**Review by mathetake — APPROVED** [review]

**Pull request merged**